pkgs/tools/admin/exoscale-cli/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, buildGo114Package, substituteAll, fetchgit }:
+
+let
+  version = "1.12.0";
+  rev = "4b82b898b56ea238eb80ac38da40dc8a460a7de7";
+in
+buildGo114Package {
+  name = "exoscale-cli-${version}";
+
+  src = fetchgit {
+    url = "https://github.com/exoscale/cli.git";
+    rev = "${rev}";
+    sha256 = "04ym7mfv565icj3lmd2nrvq9asawwmmzg09681pj9py61ws56bxr";
+    fetchSubmodules = true;
+  };
+
+  goPackagePath = "github.com/exoscale/cli";
+  subPackages = [ "." ];
```

Comment:
Isn't it the default?

Reply:
Doesn't look like it - if I remove this it dumps other binaries (doc) in the resulting `bin/`

Reply:
Yea I encountered such issue as well once... I'd add a comment right above this that explains this. What I did back when I encountered this is:

https://github.com/NixOS/nixpkgs/blob/fa0849d02308ea4d0b646f6fffff07cd042ab810/pkgs/servers/gotify/default.nix#L41-L43